### ISSUE TITLE
Talos for regression problem

### ISSUE THREAD

**Issue by mertensu** [autonomio/talos#106]
I wonder if Talos can be used for regression problems as well. I would like to simply monitor the validation loss for each combination of hyperparameters. When I call `ta.Scan`, I get an error about `predict_classes` (becauce I have a functional model) but even when I use the dev version, Talos is not doing what I expect since it still assumes a classification problem.

Thanks in advance for your help

**Comment by mikkokotila** [comment]
Thanks for posting this. For functional model, have you tried `ta.Scan(functional_support=True)` already? 

Regarding regression problems, I will do some testing today and report back. Should definitely have this resolved no later than tomorrow, it's critical that we support regression problems as much as classification problems (even thought they are generally rarer).

**Comment by mikkokotila** [comment]
Sorry for the delay. This seems to be now done in the regression branch. It's been tested heavily (some tens of thousands of permutations) using the Kaggle House Pricing dataset. Hopefully I can get it to production so it's available through pip as well. 

Thanks again for flagging this.
